CHANGELOG.md

```diff
@@ -6,6 +6,10 @@
 
 Removed the `-by_range` flag.
 
+### Updated dependencies
+
+ * trillian from v1.3.11 to v1.3.12-0.20210205094822-acd1b78e0a24
```

Comment:
Is this true? :)

Reply:
Well, it was true six weeks ago, when I sent it for review... 😉